pkgs/servers/corosync/default.nix

```diff
@@ -1,50 +1,61 @@
 { stdenv, fetchurl, makeWrapper, pkgconfig, nss, nspr, libqb
-, dbus ? null
-, librdmacm ? null, libibverbs ? null
-, libstatgrab ? null
-, net_snmp ? null
+, dbus, librdmacm, libibverbs, libstatgrab, net_snmp
+, enableDbus ? false
+, enableInfiniBandRdma ? false
+, enableMonitoring ? false
+, enableSnmp ? false
```

Comment:
You can also enable features by default, if you think they are interesting for the general user of this package.